<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SquareInfoWidget</class>
 <widget class="QWidget" name="SquareInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Square Info</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Defs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Definitions Square</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Def 1</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Def1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Def 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Def2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>72</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>73</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Def1_Pos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LEFT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TOP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BOTTOM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Def1_Dir">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LEFT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DOWN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Def2_Pos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LEFT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TOP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BOTTOM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>212</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>213</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Def2_Dir">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LEFT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DOWN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox_ncasillas1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ncasillas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ncasillas</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_ncasillas2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
